homeassistant/components/combined_energy/coordinator.py

```diff
@@ -0,0 +1,136 @@
+"""DataUpdateCoordinator for the PVOutput integration."""
+from __future__ import annotations
+
+from abc import abstractmethod
+from datetime import timedelta
+from typing import Generic, TypeVar
+
+from combined_energy import CombinedEnergy
+from combined_energy.exceptions import CombinedEnergyAuthError, CombinedEnergyError
+from combined_energy.helpers import ReadingsIterator
+from combined_energy.models import ConnectionStatus, DeviceReadings
```

Comment:
make these local imports
from .
from .exceptions

Reply:
`combined_energy` is not a local package. It is the external async client library for combined energy.
https://github.com/timsavage/combined-energy-api

Reply:
Ah, my bad, I took too quick of a glance at it.